ux/damage_panel.go

```diff
@@ -43,17 +44,56 @@ func NewDamagePanel(entity *gurps.Entity) *DamagePanel {
 	})))
 	p.DrawCallback = func(gc *unison.Canvas, rect unison.Rect) { drawBandedBackground(p, gc, rect, 0, 2, nil) }
 
+	// Add basic thrust damage
 	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
 		f.SetTitle(p.entity.Thrust().String())
 		MarkForLayoutWithinDockable(f)
 	}))
 	p.AddChild(NewPageLabel(i18n.Text("Basic Thrust")))
 
+	// Add basic swing damage
 	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
 		f.SetTitle(p.entity.Swing().String())
 		MarkForLayoutWithinDockable(f)
 	}))
 	p.AddChild(NewPageLabel(i18n.Text("Basic Swing")))
 
+	// Add Lifting Strength-based damage
+	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
+		f.SetTitle(p.entity.LiftingThrust().String())
+		MarkForLayoutWithinDockable(f)
+	}))
+	p.AddChild(NewPageLabel(i18n.Text("Lifting Damage")))
+
+	// Add IQ-based damage
+	if entity.SheetSettings.ShowIQBasedDamage {
+		p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
+			iqST := entity.ResolveAttributeCurrent("iq")
+			iqDamage := entity.ThrustFor(fxp.As[int](iqST)).String()
+			f.SetTitle(iqDamage)
+			MarkForLayoutWithinDockable(f)
+		}))
+		p.AddChild(NewPageLabel(i18n.Text("IQ-Based Damage")))
+	}
+
+	// Add TK Strength-based damage
+	tkST := entity.TelekineticStrength()
+	if tkST > 0 {
+		p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
+			tkThrust := p.entity.ThrustFor(fxp.As[int](tkST)).String()
+			f.SetTitle(tkThrust)
+			MarkForLayoutWithinDockable(f)
+		}))
+		p.AddChild(NewPageLabel(i18n.Text("TK Thrust")))
+
+		// Add TK Strength-based damage
```

Comment:
Either remove this comment since it is redundant with the one above, or specify that it is for swing and change the other to mention thrust.